src/ad/administration/adStatus/AdStatusEdit.less

```diff
@@ -28,6 +28,10 @@
         }
     }
 
```

Comment:
Knappen blir veldig dominerende om den kommer i hele bredden, når de to knappene over deler på en bredde.